src/Dev/Constraint/SSListContains.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace SilverStripe\Dev\Constraint;
+
+use PHPUnit_Framework_ExpectationFailedException;
+use SilverStripe\Dev\SSListExporter;
+use SilverStripe\View\ViewableData;
+
+/**
+ * Constraint for checking if a SS_List contains items matching the given
+ * key-value pairs.
+ *
+ * Class SSListContainsOnly
+ * @package SilverStripe\Dev\Constraint
+ */
+class SSListContains extends \PHPUnit_Framework_Constraint
```

Comment:
If phpunit isn't installed this code may run the risk of breaking a production environment due to the manifest builder.

Any code outside of the `tests` folder is pretty much at risk of this.

See the fixes we added to SapphireTest that protect against this:

 - Add `TestOnly` interface to all phpunit-dependant classes. This will need to be on all classes down the hierarchy, in case you subclass this. :)
 - Add `if (!class_exists(PHPUnit_Framework_Constraint::class)) { return; }` before the class definition.